src/frameworks/uikit/ui_color.rs

```diff
@@ -91,6 +91,15 @@ pub const CLASSES: ClassExports = objc_classes! {
 
 // TODO: more initializers, set methods, more accessors
 
+- (id)initWithWhite:(CGFloat) w alpha:(CGFloat) a {        
+    let _w = w.clamp(0.0, 1.0);
+    let _a = a.clamp(0.0, 1.0);
```

Comment:
Thanks for the changes, it looks good now :)

I want to make a small suggestion though: in Rust, the `_` prefix is used for a value we don't care about and which won't be used (it silences “unused variable” warnings), but these variables are used. Perhaps you can do `let w` and `let a` instead of `let _w` and `let _a`. Rust lets you “shadow” the existing `w` and `a` variables, I use this a lot.

Reply:
Did you miss this comment?

Reply:
It's fine if you don't want to change this, but it doesn't seem like you changed it…